school-library/students/student.rb

```diff
@@ -0,0 +1,12 @@
+require '../person/persons'
+
+class Student < Animal
```

Comment:
- [ ] The `Student` class is supposed to inherit from the `Person` class but it is currently inheriting from the `Animal` class. Kindly fix this.